apollo-federation/src/query_plan/fetch_dependency_graph.rs

```diff
@@ -592,15 +592,11 @@ impl FetchDependencyGraphNodePath {
     }
 
     fn advance_field_type(&self, element: &Field) -> Result<IndexSet<Name>, FederationError> {
-        if !element
```

Comment:
Bugs:
- We shouldn't be silently ignoring errors here when getting the output base type.